homework/index.js

```diff
@@ -44,7 +105,7 @@
         return;
       }
       const ul = createAndAppend('ul', root);
-      repos.forEach(repo => renderRepoDetails(repo, ul));
+      repos.sort(sortElements).forEach(repo => renderRepoDetails(repo, ul));
```

Comment:
Since the function body of  `sortElements` function is just a single line, you might as well code it in-line using an arrow function:

```js
repos
  .sort((a, b) => a.name.localeCompare(b.name))
  .forEach(repo => renderRepoDetails(repo, repoContainer));
```